src/bitcoin.udl

```diff
@@ -1,7 +1,45 @@
 namespace bitcoin {};
 
+// ------------------------------------------------------------------------
+// Core types
```

Comment:
I was thinking it would mirror the soon-to-be primitives in `rust-bitcoin`. I don't think we will be able to follow them exactly but if you think that makes sense I would be happy labeling it "Primitive types" or even just "Types" or "Structs"

Reply:
Yeah core types works for now. It's true that we might end up combining a few things in here (rust-miniscript being the first one) so in the end it might be good to separate them by crate name sortofish